### DISCUSSION TITLE
Run JTL on AWS with https and self signed cert

### DISCUSSION THREAD

**Discussion by data-source** [ludeknovy/jtl-reporter#170] [Q&A] [answered]
Hi, 

Is there a way to run it with https and self signed cert with current configuration or does it need to be implemented another way? 

Thanks

**Comment by ludeknovy** [comment, marked as answer]
Hi @data-source !
Currently there is no way, but I've created a task for it some time ago: https://github.com/ludeknovy/jtl-reporter/issues/128

The idea was, if you provide the cert in config, it should try to load it and use it.

**Reply by data-source** [reply]
Hi @ludeknovy ! 

Thanks for the quick reply!
That'd be a great enhancement!!

**Reply by data-source** [reply]
@ludeknovy  Was able to do an httpd conf with self-signed cert, works perfect, will do a doc on it soon enough

**Reply by ludeknovy** [reply]
Great! Looking forward to it!

**Reply by data-source** [reply]
*Think* I tried to update with latest SSL guide but not sure I haven't made a mess of it...
